### PR TITLE
Feature/cab 2668

### PR DESCRIPTION
**Pull request by agagne**

### CODE REVIEW

File: src/app/content/edit-page/edit-page.component.html [hunk 1]
```diff
@@ -13,7 +13,8 @@
           [contentItem]="contentItem"></app-content-pager>
       </span>
     </mat-toolbar>
-    <form [formGroup]="form">
+    <h2>{{pageConfig.pageName}}</h2>
+    <form [formGroup]="form" role="main">
```
Comment (maximede) [inline src/app/content/edit-page/edit-page.component.html]: `role="main"` is set on the main `div` in the `create-page-component`. Not sure what the best place is but these 2 pages should probably set up in the same way

File: src/app/search/search-box/search-box.component.html [hunk 2]
```diff
@@ -1,15 +1,15 @@
-<div class="cs-search-box">
+<div class="cs-search-box" role="search">
     <div class="search-form-group">
-        <label *ngIf="pageConfig?.searchConfig?.label">{{pageConfig.searchConfig.label}}</label>
-        <mat-form-field>
-            <input matInput [(ngModel)]="searchModel.stringQuery" (search)="updateSearch()" (ngModelChange)="updateSearch()" placeholder="{{pageConfig?.searchConfig?.placeholder}}" type="search">
+        <label for="search-field" *ngIf="pageConfig?.searchConfig?.label">{{pageConfig.searchConfig.label}}</label>
+        <mat-form-field >
+            <input id="search-field" matInput [(ngModel)]="searchModel.stringQuery" (search)="updateSearch()" (ngModelChange)="updateSearch()" placeholder="{{pageConfig?.searchConfig?.placeholder}}" type="search">
```
Comment (maximede) [inline src/app/search/search-box/search-box.component.html]: should this be a `name` instead of `id` ?

Comment (agagne) [reply src/app/search/search-box/search-box.component.html]: No, for references an element id.

File: src/app/content/edit-page/edit-page.component.html (hunk 1 above)
Comment (agagne) [reply src/app/content/edit-page/edit-page.component.html]: good catch, i've put them both in the form that contains all the the features of the page

**Pull request merged**